### ISSUE TITLE
Error for self-signed certificate

### ISSUE THREAD

**Issue by thelazyoxymoron** [go-vikunja/app#26]
Currently Vikunja expects server address to be in a particular format. It should always be `https://subdomain.domain.tld/` (ending it with slash is important).

It should also accept below combinations:

- `subdomain.domain.tld`
- `https://subdomain.domain.tld`

When used with `https://subdomain.domain.tld` (without the trailing slash), it throws this error:

![vikunja-error](https://user-images.githubusercontent.com/9106868/214014902-7782bb07-5817-4230-a0ef-9dce781e5c3b.jpg)

**Comment by kolaente** [comment]
@Benimautner can you take a look?

**Comment by Benimautner** [comment]
I'd argue that removing the need to specify the protocol (http or https) is a security risk. A compromise would be a dropdown to select either http or https.
I cannot reproduce the error with the trailing slash on `https://try.vikunja.io` . For me, all of the options below work:

- `https://try.vikunja.io`
- `https://try.vikunja.io/`
- `https://try.vikunja.io/api/v1`
- `https://try.vikunja.io/api/v1/`

Could you share some more information about your instance? Are you using a self-signed certificate?

**Comment by thelazyoxymoron** [comment]
Alas, it seems I had overlooked a simple nginx redirection for base `/`.

My setup is: docker compose with Nginx Proxy Manager (NPM) where NPM generates LetsEncrypt certificates for each of the proxy hosts. All I needed to do was to add a base `/` location redirection in nginx config.

Documenting it in case someone stumbles across the same issue. Refer to the below screenshot:

![image](https://user-images.githubusercontent.com/9106868/214495023-ed3fe2b4-1966-4e1d-aaac-77b51f15013b.png)
